src/binary-encoding.adoc

```diff
@@ -930,23 +1101,24 @@ Unlike the short-form encoding, the long-form encoding reserves:
 If the timestamp's length is greater than or equal to `8`, it has fractional seconds that are encoded as a
 `(coefficient, exponent)` pair, similar to a <<decimals, decimal>>. However, it is illegal for the fractional
 seconds value to be greater than or equal to `1.0` or less than `0.0`. For this reason, both the coefficient and
-the exponent are encoded using unsigned types. The included coefficient `VarUInt` is unsigned to prevent the encoding of
+the exponent are encoded using unsigned types. The included coefficient `FlexUInt` is unsigned to prevent the encoding of
 fractional seconds less than `0.0`. The exponent `FixedUInt` is implicitly negative, discouraging the encoding of
 decimal numbers greater than `1.0`. Note that validation is still required; namely:
```

Comment:
What is interesting, is what we're calling an implicitly negative exponent, is what other people would call *scale* and not only that, but our restrictions on the *coefficient*/*scale* are what, in other contexts, would be called a *mantissa*.  I am not sure that we need to introduce mantissa (fractional seconds is probably good enough), but we might consider calling this negative exponent *scale*.

Reply:
I'm ok with renaming those, but I'd like to leave that for a future change as I'll need to read up on the terminology.

Reply:
Added #221.